test/e2e/specs/plugins/plugins__search.ts

```diff
@@ -20,10 +20,39 @@ describe( DataHelper.createSuiteTitle( 'Plugins search' ), function () {
 	it( 'Visit plugins page', async function () {
 		pluginsPage = new PluginsPage( page );
 		await pluginsPage.visit();
+		if ( envVariables.VIEWPORT_NAME !== 'mobile' ) {
+			// Ensure the page is wide enough to show the breadcrumb details.
+			await page.setViewportSize( { width: 1300, height: 1080 } );
```

Comment:
The non-mobile viewport size is not large enough to render the full-screen breadcrumbs on the plugin details page. So I bumped the width up here.

Reply:
Thanks, I have to make same change on other tests

Reply:
**question**: what does this mean for users that are on smaller viewports where the width is not 1300px?

Reply:
I believe the breakpoint is actually at 1280, but on screens smaller that 1280, the breadcrumbs collapse into a "Back" button.
![breadcrumbs-collapse](https://user-images.githubusercontent.com/140841/187551293-8a681e93-e9f1-4149-a43f-6410eede4c95.gif)

Reply:
Thanks, I validated the behavior on my end as well.

In this case, I would suggest the viewport resize action happen in the `beforeAll` hook if possible.

Reply:
Makes sense, thanks! Moved to `beforeAll`.